source_control/git.py

```diff
@@ -132,6 +132,15 @@
         description:
             - Path to git executable to use. If not supplied,
               the normal mechanism for resolving binary paths will be used.
+
+    umask:
+        required: false
+        default: null
+        version_added: "2.0"
```

Comment:
No, in fact, the CI test that for new module. But that's still 2.1 (even without the test)